blocktx/peer_handler.go

```diff
@@ -447,14 +447,15 @@ func (bs *PeerHandler) insertBlock(blockHash *chainhash.Hash, merkleRoot *chainh
 	return bs.store.InsertBlock(context.Background(), block)
 }
 
-func printMemStats() {
+func (bs *PeerHandler) printMemStats() {
 	bToMb := func(b uint64) uint64 {
 		return b / 1024 / 1024
 	}
 	var mem runtime.MemStats
 	runtime.ReadMemStats(&mem)
-	fmt.Printf("Alloc = %v MiB, TotalAlloc = %v MiB, Sys = %v MiB, NumGC = %v\n",
-		bToMb(mem.Alloc), bToMb(mem.TotalAlloc), bToMb(mem.Sys), mem.NumGC)
+	bs.logger.Info(fmt.Sprintf("Alloc = %v MiB, TotalAlloc = %v MiB, Sys = %v MiB, NumGC = %v\n",
```

Comment:
I'm not sure it's worth converting the function into method just to use the logger.